bikeshed/HTMLSerializer.py

```diff
@@ -1,9 +1,9 @@
 # -*- coding: utf-8 -*-
 
 from __future__ import division, unicode_literals
-import re
 import StringIO
 from .htmlhelpers import childNodes, isElement, outerHTML, escapeHTML, escapeAttr
+from .messages import die
```

Comment:
Everywhere I use messages, I import the whole set with `import *`. No sense being different here.